website/contributing/release-candidate-minor.md

```diff
@@ -0,0 +1,170 @@
+
+```markdown
+<!-- Template for pre-release GitHub release -->
+
+- <!-- TODO List out notable picks for this patch -->
+
+---
+
+To test it, run:
+
+<!-- TODO Update with your version -->
+
+npx react-native init RN067RC5 --version 0.67.0-rc.5
+
+---
+
+You can participate in the conversation on the status of this release in the [working group](https://github.com/reactwg/react-native-releases/discussions).
+
+---
+
+To help you upgrade to this version, you can use the [upgrade helper](https://react-native-community.github.io/upgrade-helper/) ⚛️
+
+---
+
+See changes from this release in the [changelog PR](https://github.com/facebook/react-native/labels/%F0%9F%93%9D%20Changelog)
+```
+
+<figure>
+  <img width="400" alt="Creating a GitHub Release" src="https://user-images.githubusercontent.com/1309636/133348648-c33f82b8-b8d2-474a-a06e-35a1fb8d18de.png"/>
+  <figcaption>Creating a GitHub Release.</figcaption>
+</figure>
+
+### 4. Create a tracking discussion post
+
+Create a "Road to <YOUR_MINOR_VERSION>" discussion post in [`react-native-releases`](https://github.com/reactwg/react-native-releases/discussions):
+
+```markdown
+<!-- Template for a new minor release candidate -->
+<!-- Title: Road to <YOUR_VERSION> -->
+
+The branch cut has happened.
+
+## Notice
+
+<!-- TODO update the version -->
+
+- [Current release candidate: 0.68.0-rc.0][current-release]
+- Have an issue with current release candidate? [File an issue][issue-form] and we will triage.
+- Have a pick request for this release? Does it fall under our [pick request qualifications][release-faq]? If so please create a PR against the release branch and comment with the PR link
+- If you are release testing. Copy and fill [Test Checklist](/contributing/release-testing#test-checklist).
+
+#### Highlighted Changes in this release
+
+<!-- Add stand-out changes in this release, and link to changelog PR.  -->
+
+- Checkout this [Changelog PR][changelog-pr]
+
+## [Release Process][release-processes]
+
+#### Checklist
+
+- [ ] [Changelog PR][changelog-pr]
+- [ ] Start a Google doc of blog post for release and invite contributors of release highlights to expand
+- [ ] Follow up on [release dependencies][release-dependencies]
+  > When ready to publish stable
+- [ ] Ship changelog
+- [ ] Ship blog post
+- [ ] Notify `react-native-website` to ship new version
+
+#### Retrospective Topics
+
+<!-- List out pain points, issues to investigate that are not release-blocking to follow up on -->
+
+-
+
+## Release Status
+
+### Tracking 0.67.0-rc.1
+
+#### Blocking issues for releasing 0.67.0-rc.1
+
+-
+
+#### Picks for 0.67.0-rc.1
+
+-
+
+[changelog-pr]: https://github.com/facebook/react-native/labels/%F0%9F%93%9D%20Changelog
+[current-release]: https://github.com/facebook/react-native/releases
+[changelog-wiki]: https://github.com/facebook/react-native/wiki/Release-Changelog
```

Comment:
```suggestion
[changelog-wiki]: https://github.com/facebook/react-native/wiki/Release-Changelog
```

It looks like this wiki page do not exist anymore, and link leads to wiki home. Any idea if this has this been moved somewhere else @lunaleaps? Maybe releases WG repo?

Reply:
This what is the message I see when I try to access the link, which is a bit confusing:
<img width="343" alt="Screenshot 2022-03-08 at 13 16 31" src="https://user-images.githubusercontent.com/719641/157236616-31a1f4d8-dd2c-4b52-a5d9-fd18a698ae2e.png">